assertj-core/src/test/java/org/assertj/core/error/ShouldContainSubsequenceOfCharSequence_create_Test.java

```diff
@@ -57,10 +60,30 @@ void should_create_error_message_with_custom_comparison_strategy() {
     // THEN
     then(message).isEqualTo(format("[Test] %nExpecting actual:%n" +
                                    "  \"" + actual + "\"%n" +
-                                   "to contain the following CharSequences in this order:%n" +
+                                   "to contain the following CharSequences in this order (possibly with other values between them):%n"
+                                   +
                                    "  [\"{\", \"author\", \"title\", \"}\"]%n" +
                                    "but \"title\" was found before \"author\"%n" +
                                    "when comparing values using CaseInsensitiveStringComparator"));
   }
 
+  @Test
+  void should_create_error_message_indicating_duplicate_subsequence() {
+    // GIVEN
+    String[] sequenceValues = { "{", "title", "author", "title", "}" };
+    String actual = "{ 'title':'A Game of Thrones', 'author':'George Martin'}";
+    ErrorMessageFactory factory = shouldContainSubsequence(actual, sequenceValues, mapOf(entry("title", 1)),
+                                                           new ComparatorBasedComparisonStrategy(CaseInsensitiveStringComparator.INSTANCE));
+    // WHEN
+    String message = factory.create(new TextDescription("Test"), new StandardRepresentation());
+    // THEN
+    then(message).isEqualTo(format("[Test] %nExpecting actual:%n" +
+                                   "  \"" + actual + "\"%n" +
+                                   "to contain the following CharSequences in this order (possibly with other values between them):%n"
+                                   +
+                                   "  [\"{\", \"title\", \"author\", \"title\", \"}\"]%n" +
+                                   "But%n" +
+                                   "2nd occurrence of \"title\" was not found%n" +
```

Comment:
why do we go the next line ? I feel it would be better like this (unless I'm missing something) 
```
 "But the 2nd occurrence of \"title\" was not found%n" +
```
I'm adding `the` before 2nd as it sounds to me to be better English wise

Reply:
Hey Joel, thanks for all the feedback!
The idea behind the next line was (and I should have made that clean with an explicit test) if there are multiple duplicates in they would all be listed underneath each other and form a "block".
E.g.:
```
    String[] sequenceValues = { "{", "title", "George", "title", "title", "George", "}" };
    String actual = "{ 'title':'A Game of Thrones', 'author':'George Martin', 'title':'The Kingkiller Chronicle', 'author':'Patrick Rothfuss'}";
```
The expected outcome (with newline) would be something like:
```
"But"
"the 3rd occurrence of \"title\" was not found%n"
"the 2nd occurrence of \"George\" was not found%n"
```

Without the new line:
```
"But the 3rd occurrence of \"title\" was not found%n"
"the 2nd occurrence of \"George\" was not found%n"
```

But this can be easily adapted to you preference!
Or should this scenario handled in another way?
For example, fail with the first duplicate encountered?

Reply:
Thanks for the clarification on the wording, I would say it is better to report as many errors as possible and for the wording my preference would go to this:
```
But: 
- the 3rd occurrence of \"title\" was not found
- the 2nd occurrence of \"George\" was not found
```
and if there is only one error: 
```
But the 3rd occurrence of \"title\" was not found
```